task_3_part_1.py

```diff
@@ -0,0 +1,15 @@
+soglasnye = ['b', 'c', 'd', 'f', 'g', 'h', 'k', 'l', 'm', 'n', 'p', 'r', 's', 't', 'w', 'v', 'x', 'z']
```

Comment:
These letters must be stored in one of mutable data types (str() or tuple())
Example:
`"bcdfghjklmnpqrstvwxyz" or (b', 'c', 'd', 'f', 'g', 'h', 'k', 'l', 'm', 'n', 'p', 'r', 's', 't', 'w', 'v', 'x', 'z')`